package.json

```diff
@@ -68,10 +69,14 @@
     "fse": "^4.0.1",
     "husky": "^3.0.9",
     "jest": "^24.9.0",
+    "mini-css-extract-plugin": "^0.8.0",
```

Comment:
Pre-merge - once the Webpack config is settled for setting this up we can pull this back up a level to the Webpack starter.